draft-ietf-quic-qlog-quic-events.md

```diff
@@ -1076,25 +1078,17 @@ TransportDataMoved = {
     ? from: "user" / "application" / "transport" / "network" / text
     ? to: "user" / "application" / "transport" / "network" / text
 
-    ; raw bytes that were transferred
-    ? data: hexstring
+    ? raw: RawInfo
 }
 ~~~~
 {: #transport-datamoved-def title="TransportDataMoved definition"}
 
-Note: we do not for example use a "direction" field (with values "up" and "down")
+This event does not utilize a "direction" field (with values "up" and "down")
 to specify the data flow. This is because in some optimized implementations, data
 might skip some individual layers. Additionally, using explicit "from" and "to"
 fields is more flexible and allows the definition of other conceptual "layers"
 (for example to indicate data from QUIC CRYPTO frames being passed to a TLS
-library ("security") or from HTTP/3 to QPACK ("qpack")).
-
-Note: this event type is part of the "transport" category, but really spans all
-the different layers. This means we have a few leaky abstractions here (for
-example, the stream_id or stream offset might not be available at some logging
-points, or the raw data might not be in a byte-array form). In these situations,
-implementers can decide to define new, in-context fields to aid in manual
-debugging.
+library or from HTTP/3 to QPACK).
```

Comment:
Is this paragraph necessary? There are no events with a `direction` field. The purpose of from/too fields is already desdcribed in the earlier paragraphs. This text seems superflous to me.